sympy/utilities/tests/test_debug_logs.py

```diff
@@ -0,0 +1,41 @@
+"""
+A way to check if debugging works.
+"""
+from __future__ import print_function
+import subprocess
+import sys
+import os
+import inspect
+from os.path import abspath, dirname, join, normpath
+from contextlib import contextmanager
+
+@contextmanager
+def environ(env):
+    """
+    set environment variables
+    :param env: environment variable to modify
+    """
+    original_environ = os.environ.copy()
+    os.environ.update(env)
```

Comment:
It isn't necessary to change the environment variables in the parent process. See the `env` parameter used in subprocess APIs:
https://docs.python.org/3/library/subprocess.html#using-the-subprocess-module